index.js

```diff
@@ -66,11 +73,20 @@ conventionalRecommendedBump({
   var newVersion = pkg.version
   if (!argv.firstRelease) {
     newVersion = semver.inc(pkg.version, release.releaseType)
+
+    if (argv.test) {
```

Comment:
rather than having special logic that prints this message; I'd prefer that we build `--dry-run` such that it runs an identical code path, but mocks the git and fs logic It would also be very cool if we used something like `chalk` to print some pretty output to the terminal, indicating what will go in the CHANGELOG.md.